src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/ReleaseApprovalServiceTests.cs

```diff
@@ -1280,6 +1359,240 @@ await using (var context = InMemoryApplicationDbContext(contextId))
+                    mock.GetContentBlocks<HtmlBlock>(release.Id))
+                .ReturnsAsync(new List<HtmlBlock>());
+
+            preReleaseUserService
+                .Setup(mock => mock.SendPreReleaseInviteEmail(
+                    It.Is<Release>(r => r.Id == release.Id),
+                    invite1.Email,
+                    true))
+                .ReturnsAsync(new BadRequestResult());
+
+            preReleaseUserService
+                .Setup(mock => mock.SendPreReleaseInviteEmail(
+                    It.Is<Release>(r => r.Id == release.Id),
+                    invite2.Email,
+                    true))
+                .ReturnsAsync(Unit.Instance);
+
+            preReleaseUserService
+                .Setup(mock => mock.MarkInviteEmailAsSent(
+                    It.Is<UserReleaseInvite>(i => i.Email == invite2.Email)))
+                .Returns(Task.CompletedTask);
+
+            var nextReleaseDateEdited = new PartialDate { Month = "12", Year = "2000" };
+
+            await using (var context = InMemoryApplicationDbContext(contextId))
+            {
+                var releaseService = BuildService(contentDbContext: context,
+                    releaseChecklistService: releaseChecklistService.Object,
+                    contentService: contentService.Object,
+                    preReleaseUserService: preReleaseUserService.Object);
+
+                var result = await releaseService
+                    .CreateReleaseStatus(
+                        release.Id,
+                        new ReleaseStatusCreateRequest
+                        {
+                            ApprovalStatus = ReleaseApprovalStatus.Approved,
+                            InternalReleaseNote = "Test note",
+                            PublishMethod = PublishMethod.Scheduled,
+                            PublishScheduled = "2051-06-30",
+                            NextReleaseDate = nextReleaseDateEdited
+                        }
+                    );
+
+                VerifyAllMocks(contentService,
+                    preReleaseUserService,
+                    releaseChecklistService);
+
+                result.AssertLeft();
+            }
+
+            await using (var context = InMemoryApplicationDbContext(contextId))
+            {
+                var saved = await context
+                    .Releases
+                    .HydrateReleaseForChecklist()
+                    .Include(r => r.ReleaseStatuses)
+                    .SingleAsync(r => r.Id == release.Id);
+
+                // Assert that the failure to send emails prevented the Release from completing approval.
+                // The Release should remain unchanged from the original Release, as changes to it were rolled back.
+                // Additionally, no new ReleaseStatus entries were added.
+                saved.AssertDeepEqualTo(release);
+
+                // Futhermore, we have proven that the Publisher was not informed of the Release change, as it
```

Comment:
There's a typo here

Reply:
Hehe took me about a minute to see it - sorted ta.